modules/installation/proc-install-rhdh-airgapped-full-k8s-helm.adoc

```diff
@@ -0,0 +1,234 @@
+[id="proc-install-rhdh-airgapped-full-k8s-helm_{context}"]
+= Installing {product} on a supported Kubernetes platform in a fully disconnected environment with the Helm chart
+
+In environments without internet access, a fully disconnected installation ensures that {product} can run reliably without external dependencies. This approach involves mirroring images and transferring them manually to the air-gapped environment.
+
+.Prerequisites
+
+* Skopeo 1.17 or later
+* Yq 4.4 or later
+* Access to registry.redhat.io for pulling images by using Skopeo login
+* Access to the Kubernetes cluster with `kubectl` configured
+* Helm 3.13 or later installed on the air-gapped host
+
+.Procedure
+
+. In a terminal, fetch the Helm charts values by running the following commands:
```

Comment:
Clarify where you run the command (the command is expected to fail on the air-gapped host):


```suggestion
. On the mirroring host, in a terminal, fetch the Helm charts values by running the following commands:
```